installer/templates/phx_single/config/test.exs

```diff
@@ -23,3 +23,7 @@ config :phoenix, :plug_init_mode, :runtime<%= if @html do %>
 # Enable helpful, but potentially expensive runtime checks
 config :phoenix_live_view,
   enable_expensive_runtime_checks: true<% end %>
+
+# Sort query params output of verified routes for robust url comparisons
+config :phoenix,
+  sort_verified_routes_query_params: true
```

Comment:
We'll need the same in the umbrella config, see https://github.com/phoenixframework/phoenix/blob/main/installer/templates/phx_umbrella/config/test.exs